src/shared/logger/activation.ts

```diff
@@ -116,8 +117,10 @@ export function makeLogger(
     // if we're outputting to an output channel, any other output doesn't need ANSI color codes since we have a better display from them in the IDE
     // don't alter logfile output for now since that should be more diagnostic. On the fence about this...
     const stripAnsi = opts.useDebugConsole || false
-    for (const logPath of opts.logPaths ?? []) {
-        logger.logToFile(logPath)
+    if (!isInBrowser()) {
```

Comment:
A future commit in this PR removes this logic and will log to the file (but does it differently if we are in a browser)